kratos/utilities/exact_mortar_segmentation_utility.cpp

```diff
@@ -42,7 +42,7 @@ bool ExactMortarIntegrationUtility<2,2, false>::GetExactIntegration(
     // First look if the edges of the slave are inside of the master, if not check if the opposite is true, if not then the element is not in contact
     for (unsigned int i_slave = 0; i_slave < 2; ++i_slave)
     {
-        const array_1d<double, 3>& normal = OriginalSlaveGeometry[i_slave].GetValue(NORMAL);
+        const array_1d<double, 3>& normal = OriginalSlaveGeometry[i_slave].FastGetSolutionStepValue(NORMAL);
```

Comment:
you can not change this as such... it is in the core and it may be used by other people.

Is there a test in the core to check this btw?

Reply:
There are a very simple python test in the core. The cpp tests and the most advances tests are in my application because depend of the search (that is in my application)

Reply:
then you should make a cpp test which does not depend on that and put it in the core... otherwise it does not make sense to have it there